packages/http/fetch/src/kiotaClientFactory.ts

```diff
@@ -1,16 +0,0 @@
-import type { Middleware } from "./middlewares/middleware";
```

Comment:
To be addressed in https://github.com/microsoft/kiota-typescript/issues/985 I believe